src/main/webapp/authentication-app/src/main-components/SignInPage.js

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from "react";
```

Comment:
One of the motivations you as a group mentioned for using React was its testing framework, as I remember - the auth logic should have tests.